Modules/Scripted/DICOMLib/DICOMSendDialog.py

```diff
@@ -39,8 +39,16 @@ def open(self):
     self.protocolSelectorCombobox = qt.QComboBox()
     self.protocolSelectorCombobox.addItems(["DIMSE","DICOMweb"])
     self.protocolSelectorCombobox.setCurrentText(self.settings.value('DICOM/Send/Protocol', 'DIMSE'))
+    self.protocolSelectorCombobox.currentIndexChanged.connect(self.onProtocolSelectorChange)
     self.dicomFormLayout.addRow("Protocol: ", self.protocolSelectorCombobox)
 
+    self.serverAETitleEdit = qt.QLineEdit()
+    self.serverAETitleEdit.setToolTip("AE Title")
+    self.serverAETitleEdit.text = self.settings.value('DICOM/Send/AETitle', 'CTK')
+    self.dicomFormLayout.addRow("AE Title: ", self.serverAETitleEdit)
+    # Enable AET only for DIMSE
+    self.serverAETitleEdit.enabled = self.protocolSelectorCombobox.currentIndex == 0
```

Comment:
```suggestion
    self.serverAETitleEdit.enabled = self.protocolSelectorCombobox.currentText == 'DIMSE'
```

It is better to avoid "magic numbers". Comparing displayed text is not very robust (when we add translation then we'll need to switch to using item UserData), but for now it should be good.